definitions/marts/looker_studio_marts/npq/ls_npq_tsf_details.sqlx

```diff
@@ -108,3 +108,5 @@ ON
   npq.school_urn = gias.string_urn
 WHERE
   school_urn IS NOT NULL
+  AND
+  npq.course_identifier <> 'npq-early-headship-coaching-offer'

```

Comment:
Looks good. Can you just double check with Nathan if we also need to exclude ASO? The two get lumped together a lot and I don't think ASO is on offer any more but I can't recall if we'd need to exclude it here as well.

Reply:
Nathan agreed to exclude ASO explicitly and I've recommitted with the additional filter.